inst/components/scss/toolbar.scss

```diff
@@ -0,0 +1,67 @@
+/* Toolbar */
+.bslib-toolbar {
+    display: flex;
+    align-items: center;
+    gap: 0.5rem;
+
+    /* ---- Toolbar options ---- */
+
+    &[data-align="left"] {
+        margin-right: auto;
+        align-items: flex-start;
+    }
+
+    &[data-align="right"] {
+        margin-left: auto;
+        align-items: flex-end;
+    }
+
+    &[data-size="sm"] {
+        font-size: 0.8rem;
+
+        & > * {
+            font-size: 0.8rem;
+        }
+    }
+
+    &[data-size="md"] {
+        font-size: 1rem;
+
+        & > * {
+            font-size: 1rem;
+        }
+    }
+
+    &[data-size="lg"] {
+        font-size: 1.25rem;
+
+        & > * {
+            font-size: 1.25rem;
+        }
+    }
+
+    /* ---- Adjustments to other elements ---- */
+
+    &,
+    &>* {
+        margin-bottom: 0 !important;
+        width: auto;
+    }
+
+    /* Ensures that inputs inherit the formatting of the toolbar and align correctly */
+    >.form-group.shiny-input-container, .form-group.shiny-input-container>* {
+        width: auto;
+        margin-bottom: 0;
+        min-width: 0;
+    }
+
+    // Ensure card headers with toolbars use flexbox for alignment
+    .card-header:has(> &) {
+        display: flex;
+        align-items: center;
+    }
+    .card-footer:has(> &) {
+        display: flex;
+        align-items: center;
+    }
```

Comment:
I'm tempted to say we take both of these out. `card_header()` is now flex by default and it's okay for a `toolbar()` inside `card_footer()` to be a normal block element that takes up a full row.

- [ ] Confirm that `align="right"` and `align="left"` still work as expected when using `card_footer(toolbar())`